### PR TITLE
Fix null cfg bug.

### PR DESCRIPTION
**Pull request by LLLXXXCCC**

### CODE REVIEW

File: src/Microsoft.NetCore.Analyzers/Core/Security/SourceTriggeredTaintedDataAnalyzerBase.cs [hunk 1]
```diff
@@ -77,6 +77,11 @@ public override void Initialize(AnalysisContext context)
                             }
 
                             ControlFlowGraph cfg = operationBlockStartContext.OperationBlocks.GetControlFlowGraph();
+                            if (cfg == null)
```
Comment (mavasani) [inline src/Microsoft.NetCore.Analyzers/Core/Security/SourceTriggeredTaintedDataAnalyzerBase.cs]: Can we create a unit test for this?

Comment (dotpaul) [reply src/Microsoft.NetCore.Analyzers/Core/Security/SourceTriggeredTaintedDataAnalyzerBase.cs]: Looks like `Test_NullCfg_NoDiagnostic` cover this.

Any performance concerns with `GetControlFlowGraph()` off the top of your head?

Comment (mavasani) [reply src/Microsoft.NetCore.Analyzers/Core/Security/SourceTriggeredTaintedDataAnalyzerBase.cs]: Oh are we creating a CFG for every operation block? We should certainly avoid that. CFG should only be computed for methods where we know will be perfoming DFA.

Comment (mavasani) [reply src/Microsoft.NetCore.Analyzers/Core/Security/SourceTriggeredTaintedDataAnalyzerBase.cs]: @dotpaul Can we create a work item/PR to ensure we create CFG lazily at the point where we know we will be executing DFA?

Comment (dotpaul) [reply src/Microsoft.NetCore.Analyzers/Core/Security/SourceTriggeredTaintedDataAnalyzerBase.cs]: I was hoping @LLLXXXCCC could fix this instance in this PR. I think this is the only place where the CFG is can be created without attempting DFA.

Comment (dotpaul) [reply src/Microsoft.NetCore.Analyzers/Core/Security/SourceTriggeredTaintedDataAnalyzerBase.cs]: I'll fix a new PR.

**Pull request merged**